venues/ICLR.cc/2018/Conference/python/reviewer-groups.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/python
+
+###############################################################################
+# ex. python reviewer-groups.py --baseurl http://localhost:3000
+#       --username admin --password admin_pw
+#
+# To be run after submission due date to create review invitations for all the papers.
+# For each paper:
+# 1) create reviewer group (reviewers for this paper)
+#           and nonReviewerGroup (folks that aren't allowed to read the review at least not yet)
+# 2) create review invitation
+###############################################################################
+
+## Import statements
+import argparse
+import sys
+import config
+from openreview import *
+
+## Argument handling
+parser = argparse.ArgumentParser()
+parser.add_argument('--baseurl', help="base url")
+parser.add_argument('--username')
+parser.add_argument('--password')
+
+args = parser.parse_args()
+
+## Initialize the client library with username and password
+client = Client(baseurl=args.baseurl, username=args.username, password=args.password)
+
+submissions = client.get_notes(invitation=config.BLIND_SUBMISSION)
+for paper in submissions:
+    paper_num = str(paper.number)
+    paperinv = config.CONF + '/-/Paper' + paper_num
+    print("Adding groups for Paper"+ paper_num)
+    paperGroup = config.CONF + '/Paper' + paper_num
+    authorGroup = paperGroup + '/Authors'
+
+    ## Reviewer group - people that can see the review invitation
+    reviewerGroup = paperGroup + '/Reviewers'
+    client.post_group(openreview.Group(
+        id=reviewerGroup,
+        signatures=[config.CONF],
+        writers=[config.CONF],
+        members=[],
+        readers=[config.CONF, config.PROGRAM_CHAIRS, config.AREA_CHAIRS],
+        signatories=[]))
+
+    ## Area Chair group -
+    areachairGroup = paperGroup + '/Area_Chair'
+    client.post_group(openreview.Group(
+        id=areachairGroup,
+        signatures=[config.CONF],
+        writers=[config.CONF],
+        members=[],
+        readers=[config.CONF, config.PROGRAM_CHAIRS, config.AREA_CHAIRS, areachairGroup],
+        signatories=[areachairGroup]))
+
+    ## update invitations with area chair and reviewer groups
+    public_comment_inv = client.get_invitation(paperinv+'/Public_Comment')
```

Comment:
what does it do this script now?

Reply:
oops, sorry. I think we should actually just remove this for now. I can work on a script for this later - it's not critical to the launch of the site